doc/api/net.md

```diff
@@ -556,10 +556,9 @@ socket and send it out over the wire when it is possible. (Internally it is
 polling on the socket's file descriptor for being writable).
 
 The consequence of this internal buffering is that memory may grow. This
-property shows the number of characters currently buffered to be written.
-(Number of characters is approximately equal to the number of bytes to be
-written, but the buffer may contain strings, and the strings are lazily
-encoded, so the exact number of bytes is not known.)
+property shows the number of characters buffered for writing. The number of
+characters is approximately equal to the number of bytes. The buffer may contain
+lazily-encoded strings so the exact number of bytes may differ.
```

Comment:
I don’t feel like the last sentence became much clearer here. How about this:

> As the buffer may contain strings whose length after encoding is not known, this number is only an approximation to the number of bytes in the buffer.

Also, the paragraph after this seems very odd; it tells people to control the write buffer by controlling the readable side?

Reply:
Yeah, I started to fix that subsequent paragraph too and felt like I was slipping into a rabbit hole so I stopped with this paragraph. 😀

Reply:
> Also, the paragraph after this seems very odd; it tells people to control the write buffer by controlling the readable side?

I think that paragraph goes with the first sentence of this paragraph? So maybe altogether, it should be like this (not yet accounting for the wording change you suggested):

```
This property shows the number of characters buffered for writing. The number of
characters is approximately equal to the number of bytes. The buffer may contain
lazily-encoded strings so the exact number of bytes may differ.

`net.Socket` has the property that `socket.write()` always works. This is to
help users get up and running quickly. The computer cannot always keep up
with the amount of data that is written to a socket. The network connection
simply might be too slow. Node.js will internally queue up the data written to a
socket and send it out over the wire when it is possible. (Internally it is
polling on the socket's file descriptor for being writable).

The consequence of this internal buffering is that memory may grow.
Users who experience large or growing `bufferSize` should attempt to
"throttle" the data flows in their program with
[`socket.pause()`][] and [`socket.resume()`][].
```

Reply:
Yeah, I think that’s better :+1:

Also, I’d drop the “Internally …” parenthetical, that doesn’t really fit into the description of `.bufferSize` at all. :)

Reply:
Parenthetical removed!

Reply:
I also changed things to be much closer to your suggested wording. PTAL!